packages/DropdownMenu/test/dropdownMenu.cypress.js

```diff
@@ -0,0 +1,26 @@
+describe("<DropdownMenu />", () => {
+  beforeEach(() => {
+    cy.visitStorybook("dropdownmenu-cypress--dropdownmenu-test");
+  });
+
+  it("should show hide dropdown content when when clicking trigger", () => {
+    cy.getByTestId("dropdown-menu__trigger").click();
+    cy.getByTestId("popover-content").should("be.visible");
+    cy.wait(100).then(() => {
+      cy.getByTestId("dropdown-menu__trigger").click();
+      cy.getByTestId("popover-content").should("not.be.visible");
+    });
+  });
+
+  it("should retain trigger focus when the dropdown is closed", () => {
+    cy.getByTestId("dropdown-menu__trigger").click();
+    cy.wait(100).then(() => {
```

Comment:
I think there is an alternative in cypress to do this and avoid wait.

Reply:
I have tried using `cy.clock()` with `cy.tick(100)` but currently it isn't working the same way as  `wait` currently, When we have addressed https://github.com/acl-services/paprika/issues/126 we may see a difference in its behaviour.